File: 05 - Simple Math/Haskell/main.hs
-- get a number
    -- get an input
    -- sending it over as a number

main :: IO()

main = do
    print "What is the first number? "
    a <- readLn :: IO Float
    print "What is the second number? "
    b <- readLn :: IO Float

    print(show a  ++ " + " ++ show b ++ " = " ++ show (a+b))
    print(show a  ++ " - " ++ show b ++ " = " ++ show (a-b))
    print(show a  ++ " * " ++ show b ++ " = " ++ show (a*b))
    print(show a  ++ " / " ++ show b ++ " = " ++ show (a/b))

File: 05 - Simple Math/C++/main.cpp
#include <iostream>

using namespace std;

int main(void){
    float n1, n2 = 0;

    cout << "What is the first number? ";
    cin >> n1;

    cout << "What is the second number? ";
    cin >> n2;

    cout << n1 << " + " << n2 << " = " << n1 + n2 << endl;
    cout << n1 << " - " << n2 << " = " << n1 - n2 << endl;
    cout << n1 << " * " << n2 << " = " << n1 * n2 << endl;
    cout << n1 << " / " << n2 << " = " << n1 / n2 << endl;

    return 0;
}
